src/store/wizardSlice.ts

```diff
@@ -37,6 +42,10 @@ type wizardState = {
     registrationType: string;
     activationKey: ActivationKeys['name'];
   };
+  details: {
+    blueprintName: ComposeRequest['image_name'] | '';
```

Comment:
We can remove the  `| ''`, the type of `ComposeRequest['image_name']` is already `string | undefined` so `''` is already taken care of.

Reply:
Actually since `ComposeRequest['image_name']` can be undefined, let's make the type just be `string` for now. So this should be: 
```
blueprintName: string;
blueprintDescription: string;
```